### PR TITLE
fix:#87 - add clear button in search bar

### LINKED ISSUE #87: Add clear button in Search Bar
Explanation:

When doing a search, and realize the changing the search has no tool that allows you to delete the field and perform a new search, you want this tool to be implemented.

### Screenshots:

image
image

Checklist:
 
 - [ ] Place X in the search text field.
 - [ ] Icon have some padding to be clickeable
 - [ ] Check that the responsive looks good.
 - [ ] Pressing the X button on the search would clear the search bar and perform the query with the empty text search.
 
References:

https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A2

### PR DESCRIPTION
**Pull request by arodriguezl9302**
### Description

Add clear button in Search Bar

If the field is blank then simply the clear button does not appear, but as soon as you write something it appears and when you click on it it clears both the field and the search variable in this case "q"

fixes #87 

#### Screenshots

image

#### Type of change

Bug fix (non-breaking change which fixes an issue)

### How to test it

1. Go to Home Page.

### Checklist:

- [x] Place X in the search text field.
- [x] Icon have some padding to be clickeable
- [x] Check that the responsive looks good.
- [x] Pressing the X button on the search would clear the search bar and perform the query with the empty text search.

### CODE REVIEW

File: src/components/SearchInput.jsx [hunk 1]
```diff
@@ -28,6 +33,13 @@ const SearchInput = ({ page, query }) => {
...
+            </svg>
```
Comment (heliomar-pena) [inline src/components/SearchInput.jsx]: Can you move it to assets or a individual component?

Comment (arodriguezl9302) [reply src/components/SearchInput.jsx]: ready @Katsu08

**Review by heliomar-pena — APPROVED** [review]
Good work

**Review by e-azocar — CHANGES_REQUESTED** [review]
When I use the search input at home page, the clean button redirect me to `/artworks?q=`

https://user-images.githubusercontent.com/61360270/163634710-8dc27023-719e-4de1-b678-b2808f7299d7.mp4

**Comment by heliomar-pena** [comment]
> When I use the search input at home page, the clean button redirect me to `/artworks?q=`
> screen-capture.mp4

If user is in artwork detail or another view, I think it shouln't be redirected too. 

Maybe add a conditional: `if user is in artworks list, then redirect to artworks list. else, just clear the input`

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Please check comments

**Comment by arodriguezl9302** [comment]
@Katsu08 @e-azocar let it just clear the search bar

**Review by heliomar-pena — APPROVED** [review]
That work good! Good job

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Pull request merged**